examples/taskruns/taskrun-order.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: tekton.dev/v1alpha1
+kind: TaskRun
+metadata:
+  name: test-steps-run-in-order
+spec:
+  taskSpec:
+    steps:
+    - image: busybox
+      cmd: bash
+      args: ['-c', 'sleep 3 && touch foo']
+    - image: busybox
+      cmd: bash
+      args: ['-c', 'ls', 'foo']
```

Comment:
this is a total nitpick and feel free to ignore me, but id personally rather see this as an end to end test than an example. to me the distinction is:

- examples: things ppl look at when they want to know how to do something
- end to end tests: things ppl dont need to look at which test our actual machinery

In knative/build everything was a yaml test since we didnt have the distinction :woman_shrugging:

Reply:
yes, makes sense, will do that